Add several new Kujira tokens

Rorcual-Nodes opened the pull request:
Added tokens to assetlist.json :

- xUSK
- xMNTA
- stATOM
- stOSMO
- wBTC.axl
- Several LP Tokens

Images for xTOKENS and LPs

JeremyParish69 commented on kujira/assetlist.json:
```diff
@@ -1155,6 +1155,1134 @@
+        "display": "xusk",
+        "symbol": "xUSK",
+        "logo_URIs": {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/ghost/xusk.png"
```
Please don't create a /ghost/ sub-directory within /images/

JeremyParish69 commented on kujira/assetlist.json:
```diff
@@ -1155,6 +1155,1134 @@
+        "base": "ibc/0306D6B66EAA2EDBB7EAD23C0EC9DDFC69BB43E80B398035E90FBCFEF3FD1A87",
+        "name": "stATOM",
+        "display": "statom",
+        "symbol": "stATOM",
```
Please define `traces` for all ibc-transferred assets

JeremyParish69 commented on kujira/assetlist.json:
```diff
@@ -1155,6 +1155,1134 @@
+        },
+        "images": [
+            {
+                "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.png",
```
Please include image_sync for all images referencing images stored in another chain's /images/ directory.

JeremyParish69 commented on kujira/assetlist.json:
```diff
@@ -1155,6 +1155,1134 @@
+        },
+        "images": [
+            {
+                "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.png",
```
Due to the different backgrounds (navy vs transparent), bow.svg and bow.png are not visually identical and must be registered under separate image objects. So only one of these can be primary and may go into logo_URIs

JeremyParish69 commented on kujira/assetlist.json:
```diff
@@ -1155,6 +1155,1134 @@
+                "exponent": 0
+            },
+            {
+                "denom": "lp cheq-mnta",
```
Please do not include spaces in the denom

Rorcual-Nodes commented:
@JeremyParish69  thank you so much for your review comments. First time submitting assets to the registry, Proceeded to rectify:

- Added `traces` for IBC denoms.
- Removed images subdirectory.
- Removed conflict on BOW images due to different background.
- Removed spaces in denoms.
- Added ìmage_sync` for third party chains.

JeremyParish69 commented on kujira/assetlist.json:
```diff
@@ -1,1160 +1,2264 @@
+            },
+            "images": [
+                {
+                    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
```
For cases where you use bow.svg, can we also define bow.png as a new image object? i.e., 
```
"images": [
  {
    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
  },
  {
    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.png"
  }
]
```

JeremyParish69 replied:
Or else remove bow.png because it is unused

Rorcual-Nodes replied:
Yes, bow.png was removed from all items on the last commit.

JeremyParish69 replied:
It was removed from the code but the file is still there under images. I'm saying we should make use of it (by specifying it as an alternative, or else remove the file from /images/)

Rorcual-Nodes replied:
Ah yeah! My bad. Removed the file. Cheers

JeremyParish69 commented:
Can you please correct the formatting back to 2 spaces? It is very difficult to compare the differences when the formatting is changed

Rorcual-Nodes commented:
Checked bow.png was removed and indentation back to 2.
